HelpSource/Classes/BelaScope.schelp

```diff
@@ -96,7 +106,6 @@ Examples::
 code::
 ~belaServer = Server.remote(\bela, NetAddr("192.168.6.2", 57110));
 ~belaServer.options.maxBelaScopeChannels = 8;
-~belaServer.reboot;
 
 // UGens: usage like .poll
 // scope the LFNoise0, and still use it to modulate a SinOsc
```

Comment:
In the lines below , what happens if one runs 

```
{ var freq = LFNoise0.kr.range(20,2000); SinOsc.ar(freq.belaScope(0)) }.play

{ var freq = LFNoise0.kr.range(20,2000) ! 4; SinOsc.ar(freq.belaScope(0)) }.play
```
?

Will they both scope to the same channel (i.e.: they are summed)? Or what else? Is there need for a `s.freeAll` in between?

Reply:
They are summed on the same channel, like it happens if you do the same on busses and Out.ar.